src/components/stories/Preview.js

```diff
@@ -131,22 +141,39 @@ function PreviewFactory(options, { dir = "ltr", theme = "light" } = {}) {
   const themeClass = `theme-${theme}`;
   document.dir = dir;
   document.body.parentNode.className = themeClass;
-  const div = document.createElement("div");
-  document.body.appendChild(div);
-  div.id = "foobar";
-  div.style.top = "100px";
-  div.style.left = "100px";
-  div.width = "100px";
-  div.height = "20px";
+
+  // NOTE: I'm hoping somebody smarter than me can find a way to add a target element 
+  // as a child of `editor-wrapper` in react and reference it in Preview or create an
+  // element that is always available and in the same location.
```

Comment:
i dont understand stories all that well -- but since this is essentially display information, maybe we could just do this with a class?